source/_docs/platform-considerations.md

```diff
@@ -36,9 +36,9 @@ In this or similar instances, consider performing larger operations locally, the
 
 ## Batched Data Export to File
 
-In Test and Live environments on plans with multiple application containers, it is difficult to test batched exports to a file. Many contrib plugins and modules are not designed to support multiple application containers. It might be possible to get data export working, but that is likely to require additional effort and custom code.
+For plans which have multiple application containers, bacthed data export to file is difficult. Many contrib plugins and modules are not well architected to support multiple application containers. It might be possible to get data export working, but is likely to require effort and custom code.
```

Comment:
@tinefin I removed "well architected" from the original PR because "architected" is not _really_ an English word, and can be confusing, especially when run through translators.

Reply:
Ah. Well, it is not just that word which has changed though. The meaning of the paragraph has changed so it is now saying something fairly different / incorrect.

Reply:
Well I can see that er added "test" and should remove that, but a lot of it was just switching from passive to active voice. I've assigned @EdwardAngert to take a pass at this to do so again, while keeping the message the same.